test/acceptance/tests/basic_test.go

```diff
@@ -53,15 +55,19 @@ func TestBasic(t *testing.T) {
 		"secure auto publish with privateEcrRepoName": {
 			secure:                 true,
 			autoPublishRegistrator: true,
-			privateEcrRepoName:     "test-ecr-repo",
+			privateEcrRepoName:     fmt.Sprintf("test-ecr-repo-%s", strings.ToLower(random.UniqueId())),
 		},
 	}
 
 	for name, c := range cases {
 		t.Run(name, func(t *testing.T) {
 			config := suite.Config()
+			if config.Arch == "arm64" && c.autoPublishRegistrator {
```

Comment:
Is this required? Can we use the ECS repo configured in test/acceptance/setup-terraform/ecr.tf?

Reply:
currently  we using registrator image in ecr to test this flow(pulling from public gallery and publishing to a private ecr repo)
this is a temporary check. after we publish the arm registrator image to ecr we can remove this check

Reply:
Is there a reason to use an image from the public gallery? It feels like we would find issues earlier by testing against the local branch.

Reply:
not totally sure. even the consul images used are from the public gallery.(i guess thats fine since we are not testing consul)
the autopublish flow needs to test pulling the image from somewhere and pushing it to a private ecr repo. so image has to there in some registry.
we arent pushing arm images to hashicorpdev in dockerhub(in build workflow) in other repos. so i havent done that here also. would you recommend doing that ?

Reply:
It would be nice to:
* Run the tests against images build from the local code.
* Publish the arm imagines everywhere the other images are published to.

Reply:
used image in the ECR repo configured in test/acceptance/setup-terraform/ecr.tf